pontoon/base/views.py

```diff
@@ -1257,16 +1258,29 @@ def request_projects(request, locale):
         return HttpResponseBadRequest('Bad Request: Non-existent projects specified')
 
     projects = ''.join('- {} ({})\n'.format(p.name, p.slug) for p in project_list)
+    user = request.user
+    user_url = request.build_absolute_uri(
+        reverse('pontoon.contributor', kwargs={'email': user.email})
+    )
 
     if settings.PROJECT_MANAGERS[0] != '':
         EmailMessage(
-            'Project request for {locale} ({code})'.format(locale=locale.name, code=locale.code),
-            'Please add the following projects to {locale} ({code}):\n{projects}'.format(
-                locale=locale.name, code=locale.code, projects=projects
+            subject='Project request for {locale} ({code})'.format(locale=locale.name, code=locale.code),
+            body=u'''
```

Comment:
Keeping it in the view for now for easier access. But if we keep adding stuff, I agree it should be moved into a template.